vuln_due_date_cleanup/readme.md

```diff
@@ -0,0 +1,53 @@
+# Remove DueDate from CVE IDs
+
+What is it for:
+- We all know that the Vulnerability Score changes over time, but not the SLA DUE_DATE.
+- Once a DUE_DATE is granted to a Vulnerability, even if the Vulnerability Score change, the DUE_DATE will not change.
+- When the Vulnerability Score change, the DUE_DATE should change and follow the SLA setting configured on your system.
+- This script is a work-around and will remove the DUE_DATE of all the Vulnerabilities within a specific Risk Meter that changed score in the last X days.
+- Every night Kenna runs an internal process that will grant a new DUE_DATE to those vulnerabilities that meet your SLA configuration.
+- So, after removing the needed DUE_DATE using this script, you might need to wait a day for the vulnerabilities to receive a new one.
+
+# Kenna is not responsible for maintaining this script or any harm that it may cause.
+
+This POC python script has two outcomes:
+- Will generate a CSV (ovsh.csv) with the score history for any open CVE within a specific Risk Meter.
```

Comment:
Sorry I didn't catch this the first time, but the code generates `opencves.csv`, not `ovsh.csv`.